src/.idea/codeStyles/Project.xml

```diff
@@ -3,9 +3,15 @@
     <JetCodeStyleSettings>
       <option name="LBRACE_ON_NEXT_LINE" value="true" />
     </JetCodeStyleSettings>
+    <codeStyleSettings language="Vue">
+      <indentOptions>
+        <option name="INDENT_SIZE" value="4" />
```

Comment:
IntelliJ started indenting my .vue files with 2 spaces instead of 4 for some reason so added this in to force shared style settings for .vue files